src/nav/style-guide.yml

```diff
@@ -1,119 +1,128 @@
+    - title: Writing great content at New Relic
+      path: /docs/style-guide/writing-strategies
+      pages: 
+        - title: Introduction to the style guide
+          path: /docs/style-guide/writing-strategies/introduction-style-guide
+        - title: Voice, tone, and style
+          path: /docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic
+    - title: Capitalization
+      path: /docs/style-guide/capitalization
+      pages:
+        - title: Capitalization
+          path: /docs/style-guide/capitalization/capitalization
+    - title: Formatting
+      path: /docs/style-guide/formatting
+      pages:
+        - title: Buttons
+          path: /docs/style-guide/formatting/buttons
+        - title: Callouts
+          path: /docs/style-guide/formatting/callouts      
+        - title: Collapsers
+          path: /docs/style-guide/formatting/collapsers
+        - title: Code references
+          path: /docs/style-guide/formatting/code-examples
+        - title: Code formatting - var and mark
+          path: /docs/style-guide/formatting/code-formatting-guidelines-var-mark
+        - title: Bold or code, not italics
+          path: /docs/style-guide/formatting/bold-or-code-not-italics
+        - title: Heading levels
+          path: /docs/style-guide/formatting/levels-headings
+        - title: Hyperlinks
+          path: /docs/style-guide/formatting/hyperlinks
+        - title: Lists
+          path: /docs/style-guide/formatting/lists
+        - title: Tables
+          path: /docs/style-guide/formatting/tables
+        - title: Formatting terminal commands
+          path: /docs/style-guide/formatting/formatting-terminal-commands
+        - title: Titles, short titles, and paths
+          path: /docs/style-guide/formatting/titles
+        - title: UI paths
+          path: /docs/style-guide/formatting/ui-paths
+    - title: Images, videos, and captions
+      path: /docs/style-guide/images
+      pages:
+        - title: Screenshots and images
+          path: /docs/style-guide/images/screenshots-images 
+    - title: Word choice
+      path: docs/style-guide/word-choice
+      pages:
+        - title: Usage dictionary
+          path: docs/style-guide/word-choice/usage-dictionary
+        - title: Pricing and billing language guidelines
+          path: /docs/style-guide/word-choice/pricing-language-guidelines
+        - title: Users and roles language guidelines
+          path: /docs/style-guide/word-choice/user-related-language-guidelines 
+    - title: Writing docs
+      path: /docs/style-guide/writing-docs
+      pages:
+        - title: Five questions
+          path: /docs/style-guide/writing-docs/five-questions-help-write-docs
+        - title: Organize your docs page
+          path: /docs/style-guide/writing-docs/organize-doc
+        - title: Docs translation
+          path: /docs/style-guide/writing-docs/docs-translation
+        - title: How to write docs
```

Comment:
Are the docs here too hidden?

Reply:
@urbiz-nr - can you say a little more about this concern? Some things can still move around, and will.

Reply:
I feel like "basic" stuff is easy to find, while more strategic/philosophical docs are under two levels. Maybe people will miss them, and I think those are the ones we'd really want them to read.

Reply:
I moved the five question and how to organize your docs pages up to the initial section, on writing strategy. The rest is very much about how to do docs things, like our editing processes, etc. It's pretty nuts and bolts about how to docs. I'll create a link on the Intro to the style guide page though.